ntoskrnl/ps/quota.c

```diff
@@ -5,6 +5,7 @@
  * COPYRIGHT:       Copyright 2005 Alex Ionescu <alex@relsoft.net>
  *                  Copyright 2007 Mike Nordell
  *                  Copyright 2021 George Bișoc <george.bisoc@reactos.org>
+ *                  Copyright 2022 Tuur Martens <tuurmartens4@gmail.com>
```

Comment:
Adding one's name under copyrights would make sense if one has made considerable amount of changes in code, so please revert.

Reply:
Sorry, did not know the etiquette of this.